src/modules/users/components/GasStation/TransactionDetails/TransactionBackToList.css

```diff
@@ -17,3 +17,11 @@
   color: var(--text);
   cursor: pointer;
 }
+
+.returnToSummary svg {
+  /*
+   * NOTE Using this to override Icon.css's vertical-align property
+   * which might be being used elsewhere.
+   */
+  vertical-align: -5px;
```

Comment:
My preference here would be to instead use flex to align this, using the button as the container, then center aligning items.

Reply:
I agree with Arren, would be good to use flex.